File: Iterative Width Planner (sgware)/src/edu/uno/ai/planning/iw/IteratedWidthPlanner.java
package edu.uno.ai.planning.iw;

import edu.uno.ai.planning.ss.StateSpacePlanner;
import edu.uno.ai.planning.ss.StateSpaceProblem;
import edu.uno.ai.planning.ss.StateSpaceSearch;

public class IteratedWidthPlanner extends StateSpacePlanner {
	
	public IteratedWidthPlanner() {
		super("IW2");
	}

	@Override
	protected StateSpaceSearch makeStateSpaceSearch(StateSpaceProblem problem) {
		return new IteratedWidthSearch(problem, 2);
	}
}

File: UnitySokoban/Assets/Scripts/Planning/IterativeWidthPlanner/IteratedWidthPlanner.cs
using Planning;
using StateSpaceSearchProject;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace IterativeWidthPlanner
{
    public class IteratedWidthPlanner : Planner<IteratedWidthSearch>
    {
        public IteratedWidthPlanner()
            : base("IW2")
        {
        }

        public override IteratedWidthSearch makeSearch(Problem problem)
        {
            return new IteratedWidthSearch((StateSpaceProblem)problem, 2);
        }
    }
}
